lib/core/test/unit/Cardano/Wallet/DB/Sqlite/TypesSpec.hs

```diff
@@ -0,0 +1,74 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Cardano.Wallet.DB.Sqlite.TypesSpec
+    ( spec
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.DB.Sqlite.Types
+    ()
+import Cardano.Wallet.Primitive.Types
+    ( EpochNo (..), SlotId (..), SlotNo (..) )
+import Data.Proxy
+    ( Proxy (..) )
+import Data.Typeable
+    ( Typeable, typeRep )
+import Data.Word.Odd
+    ( Word31 )
+import Database.Persist.Sqlite
+    ( PersistField (..) )
```

Comment:
**[suggestion: dependency management]**

Use [`PersistField`](https://hackage.haskell.org/package/persistent-2.10.4/docs/Database-Persist-Class.html#t:PersistField) from `persistent` instead

**Why**

This is the same function re-exported by `persistent-sqlite` however, using the version from `Database.Persist.Sqlite` creates a non-necessary dependency to `persistent-sqlite` whereas we are just interested in a class already available in the core of persistent.